src/ConnectedMode/Migration/Wizard/MigrationWizardWindow.xaml

```diff
@@ -3,24 +3,41 @@
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     xmlns:platformUi="clr-namespace:Microsoft.VisualStudio.PlatformUI;assembly=Microsoft.VisualStudio.Shell.15.0"
+    xmlns:vsShell="clr-namespace:Microsoft.VisualStudio.Shell;assembly=Microsoft.VisualStudio.Shell.15.0"
+    
     Title="SonarLint Connected Mode Migration"
     HorizontalAlignment="Center"
     ResizeMode="NoResize"
     SizeToContent="WidthAndHeight"
     MinHeight="400"
-    MaxHeight="600"
     MinWidth="400"
-    MaxWidth="600">
+    MaxWidth="600"
+    
+    Background="{DynamicResource {x:Static vsShell:VsBrushes.ComboBoxBackgroundKey}}"
+    FontSize="{DynamicResource {x:Static vsShell:VsFonts.Environment122PercentFontSizeKey}}"    
+    FontFamily="{DynamicResource {x:Static vsShell:VsFonts.EnvironmentFontFamilyKey}}"
+    
+    >
 
     <Window.Resources>
         <Style x:Key="Button_Style" TargetType="Button">
             <Setter Property="MinHeight" Value="25" />
             <Setter Property="MinWidth" Value="75" />
         </Style>
 
+        <Style TargetType="TextBlock">
```

Comment:
I copied most of the styling from the [Issue Visualization tool window XAML](https://github.com/SonarSource/sonarlint-visualstudio/blob/03123f51a5f22aa2bd6e347f00a33ca01e9ee44b/src/IssueViz/IssueVisualizationControl/IssueVisualizationControl.xaml#L35) then tweaked it a bit.